src/commands/install.rs

```diff
@@ -42,30 +42,55 @@ pub fn install() -> Result<()> {
     let fuel_core_bin_tarball_name = fuel_core_bin_tarball_name(&fuel_core_release_latest_tag)?;
 
     info!("Fetching forc {}", &forc_release_latest_tag);
-    download_file_and_unpack(
+
+    let mut installed_bins_message = String::new();
+    let mut errored_bins_message = String::new();
+
+    match download_file_and_unpack(
         SWAY_RELEASE_DOWNLOAD_URL,
         &forc_release_latest_tag,
         &forc_bin_tarball_name,
-    )?;
+    ) {
+        Ok(()) => installed_bins_message.push_str(&format!("forc {}", &forc_release_latest_tag)),
+        Err(_) => errored_bins_message.push_str(&format!("forc {}", &forc_release_latest_tag)),
```

Comment:
Another more concise/efficient way of writing to these strings is:

```rust
write!(&mut errored_bins_message, "forc {}", &forc_release_latest_tag);
```

Same with below